common/core/build.gradle.kts

```diff
@@ -88,7 +88,7 @@ android {
 
 apply(from = "../../android/scripts/maven-meta.gradle")
 
-val xcframeworkPath = "build/bin/xcframework/MaasCore.xcframework"
+val xcframeworkPath = "../../ios/MaasCore/Sources/MaasCore/Core.xcframework"
```

Comment:
Out of curiosity, is there any technical benefit to moving the location of the xcframework?

Reply:
When we're trying to fetch MaasCore for development we're using `.package(path: "/ios/MaasCore")` which requires as to create a target inside `Package.swift` for `MaasCore` and path for target must by inside package directory.